pkg/ddevapp/list.go

```diff
@@ -69,25 +70,20 @@ func List(settings ListCommandSettings) {
 			}
 
 			routerStatus, _ := GetRouterStatus()
-			var extendedRouterStatus = RenderRouterStatus()
-			if nodeps.ArrayContainsString(globalconfig.DdevGlobalConfig.OmitContainersGlobal, globalconfig.DdevRouterContainer) {
-				extendedRouterStatus = "disabled"
+			routerURL := globalconfig.GetRouterURL()
+			if routerStatus == SiteStopped {
+				routerURL = ""
+			}
+			extendedRouterStatus := RenderRouterStatus()
+			routerType := globalconfig.DdevGlobalConfig.Router
+			if len(types.GetValidRouterTypes()) < 2 {
+				routerType = ""
 			}
-			router := globalconfig.DdevGlobalConfig.Router
 			t.AppendFooter(table.Row{
-				"Router", routerStatus, fileutil.ShortHomeJoin(globalconfig.GetGlobalDdevDirLocation()), globalconfig.GetRouterURL(), router},
+				"Router", extendedRouterStatus, fileutil.ShortHomeJoin(globalconfig.GetGlobalDdevDirLocation()), routerURL, routerType},
 			)
 			t.Render()
 			output.UserOut.WithField("raw", appDescs).Print(out.String())
-			if routerStatus != "healthy" {
-				rawResult := map[string]string{
-					"routerStatus":         routerStatus,
-					"extendedRouterStatus": extendedRouterStatus,
-				}
-				rawResult["routerStatus"] = routerStatus
-				rawResult["extendedStatus"] = extendedRouterStatus
-				output.UserOut.WithField("raw", rawResult)
-			}
```

Comment:
This code was never executed because it lacked `Print()`:

```diff
-output.UserOut.WithField("raw", rawResult)
+output.UserOut.WithField("raw", rawResult).Print()
```

However, if we add `Print()`, it could break the parsing for `ddev list -j | jq .raw`

We cannot change syntax of the `raw` field in the original JSON output for `ddev list -j`.

One option is to add a new optional field for the JSON output, like `raw_extra`, where we can add a new `router` field.

Alternatively, we can simply remove it, which is what I did here, because it never worked.